docs/web3-eth-txpool.rst

```diff
@@ -0,0 +1,265 @@
+.. _eth-txpool:
+
+.. include:: include_announcement.rst
+
+========
+web3.eth.txpool
+========
+
+
+The ``web3-eth-txpool`` package gives you access to several non-standard RPC methods to inspect the contents of the transaction pool containing all the currently pending transactions as well as the ones queued for future processing.
+
+
+.. code-block:: javascript
+
+    import Web3 from 'web3';
+    import {Txpool} from 'web3-eth-txpool';
+
+    // "Web3.givenProvider" will be set if in an Ethereum supported browser.
+    const txpool = new Txpool(Web3.givenProvider || 'ws://some.local-or-remote.node:8546', null, options);
+
+
+    // or using the web3 umbrella package
+    const web3 = new Web3(Web3.givenProvider || 'ws://some.local-or-remote.node:8546', null, options);
+
+    // -> web3.eth.txpool
+
+
+------------------------------------------------------------------------------
+
+
+.. include:: include_package-core.rst
+
+
+
+------------------------------------------------------------------------------
+
+.. _txpool-content:
+
+content
+=========
+
+.. code-block:: javascript
+
+    web3.eth.txpool.content([callback])
+
+This API can be used to list the exact details of all the transactions currently pending for inclusion in the next block(s), as well as the ones that are being scheduled for future executions.
+The RPC method used is ``txpool_content``.
+
+----------
+Parameters
+----------
+
+1. ``Function`` - (optional) Optional callback, returns an error object as first parameter and the result as second.
+
+-------
+Returns
+-------
+
+``Object`` - The list of pending as well as queued transactions.
```

Comment:
Please dont just put `object` write the current interface it returns with full details in which field does what else users will just raise issues.